### PR TITLE
Adding election procedures as a policy document

### PR DESCRIPTION
**Pull request by ceb8**
Moving the election procedure document from Google Docs into GitHub.

### CODE REVIEW

File: policies/election-procedures.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+# Astropy Election Procedures
+
+## Overview
+
+Astropy votes are managed with the [Helios Voting platform](https://heliosvoting.org) and
+announced publicly via the astropy-dev mailing list and the OpenAstronomy Discourse forum. 
+
+## Procedure for creating an election
+
+1.  Set a date for the election and announce the election
```
Comment (pllim) [inline policies/election-procedures.md:10]: Might want to change each number into `###` sub-section as GitHub isn't rendering the `ul` listing among the numbering nicely.

File: policies/election-procedures.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+We have already received some nominations from the Roles survey, but are accepting additional
+nominations for new voting members until the end of <June 21> (anywhere on Earth).
+To nominate someone (or yourself) to be a voting member, please fill out this form:
+https://forms.gle/f1hi6MNEmw1RzYPY7 
+
+For the Coordination Committee,
+<sender name>
+
+—------------------
```
Comment (pllim) [inline policies/election-procedures.md:92]: Not sure what this divider is supposed to do.

File: policies/election-procedures.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+Subject: Announcing Astropy Voting Member Election: <July 2022>
+
+Dear Astropy Community,
+
+The next Astropy Voting Member election will be on <July 8, 2022.>
+
+We have already received some nominations from the Roles survey, but are accepting additional
+nominations for new voting members until the end of <June 21> (anywhere on Earth).
+To nominate someone (or yourself) to be a voting member, please fill out this form:
+https://forms.gle/f1hi6MNEmw1RzYPY7
```
Comment (pllim) [inline policies/election-procedures.md:87]: Is this link static or just an example?

File: policies/election-procedures.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+Astropy votes are managed with the [Helios Voting platform](https://heliosvoting.org) and
+announced publicly via the astropy-dev mailing list and the OpenAstronomy Discourse forum. 
+
+## Procedure for creating an election
+
+1.  Set a date for the election and announce the election
+   - This sets various deadlines that must be followed by nominees and voting
+      members / CoCo nominees.
+   - Election start date must be at least 4 weeks after the announcement email
```
Comment (pllim) [inline policies/election-procedures.md:13]: Consistent period at the end of all sentences throughout this doc would be nice.

**Comment by pllim** [comment]
Thanks! I left some comments here (and one private comment in private).

File: policies/election-procedures.md (hunk 3 above)
Comment (ceb8) [reply policies/election-procedures.md:87]: It's static, it's the one we've used every time and after the last time Erik moved it from his person drive into the astropy space so it should be able to just stick around.

File: policies/election-procedures.md (hunk 2 above)
Comment (ceb8) [reply policies/election-procedures.md:92]: It's between the templates.... I just copied it over from the google doc, have you got a better suggestion?

Comment (pllim) [reply policies/election-procedures.md:92]: Does HTML tag work?

```html
<hr/>
```

Would look like this:

<hr/>

File: policies/election-procedures.md (hunk 4 above)
Comment (ceb8) [reply policies/election-procedures.md:13]: Went with the no periods on bullets option.

File: policies/election-procedures.md (hunk 1 above)
Comment (ceb8) [reply policies/election-procedures.md:10]: Fixed the bullets.

**Comment by ceb8** [comment]
@pllim I've addressed all your comments (and a few more things I found while fixing things).

**Review by pllim — APPROVED** [review]
Thanks! Since this is copy pasting an existing process that is already working, I don't think we need multiple approvals to merge. We can always fine-tune later as needed.

**Pull request merged**